package.json

```diff
@@ -9,6 +9,7 @@
     "chromatic": "chromatic --project-token=e10acf0c74f9 --force-rebuild",
     "ci": "vitest --dir src",
     "ci:e2e": "playwright test",
+    "debug:ci:e2e": "PWDEBUG=1 playwright test",
```

Comment:
Can you reverse the wording here? I'd like the order to go from lower to higher specificity. So, this instead: `ci:e2e:debug`.